### ISSUE TITLE
`etcdctl member remove` should support name or node ID

### ISSUE THREAD

**Issue by robszumski** [coreos/etcd#2195]
When removing a member, the user should be able to specify the name _or_ node ID of the member to be removed.

Here's what currently happens:

```
$ ./etcdctl --peers http://localhost:2379 member remove infra0
unexpected status code 404
```

If referencing by name isn't possible for some reason, we should have a better error, since this seems like it should work (to a user) and we should handle it:

```
$ ./etcdctl --peers http://localhost:2379 member remove infra0
Error: Can not remove member by name, please use the member ID (ex. 19ac17627e3e396f)
$ ./etcdctl --peers http://localhost:2379 member remove bf9071f4639c75cc
Removed member bf9071f4639c75cc from cluster
```

**Comment by xiang90** [comment]
@robszumski name is impossible. It is not a identification and multiple member might have the same name. I think we need a better documentation around that.

**Comment by robszumski** [comment]
If that's the case we absolutely need to handle the error more gracefully, as shown above.

**Comment by robszumski** [comment]
I don't think this is a documentation issue, it already mentions to use the ID. But as a user, I expect to be able to use both.

**Comment by barakmich** [comment]
Fixed in #2201
